misc/build_helpers/_build-wheels.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Based on https://github.com/pypa/python-manylinux-demo
+
+set -e -x
+
+# Compile wheels
+for PYBIN in /opt/python/cp2*/bin; do
```

Comment:
That's what the Docker images being used have.  I don't know why they picked that path in particular but the images include a number of different Python builds (each ABI incompatible with the rest).  The output of the build is one wheel per Python ABI.